### PR TITLE
Added read json function and edited readme

### LINKED ISSUE #8: Reading json file
Because we often read configuration from a json file, I think adding a function to read the json file would be great.

### PR DESCRIPTION
**Pull request by Tscunami**
Added function for reading a json file. Also updated README file.

Closes #8

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,8 +42,11 @@ urpautils.write_txt_file("C:\\path\\to\\file.txt", "This will be written to the
 # Append to text file
 urpautils.write_txt_file("C:\\path\\to\\file.txt", "This will be appended to the file", mode="a")
 
-# Read text file. Returns None if the file does not exist
-content = read_txt_file("C:\\path\\to\\file.txt")
+# Read text file. Returns FileNotFoundError if the file does not exist
```
Comment (anerold) [inline README.md:45]: Raises FileNotFoundError, not "Returns"

File: README.md [hunk 2]
```diff
@@ -42,8 +42,11 @@ urpautils.write_txt_file("C:\\path\\to\\file.txt", "This will be written to the
...
+content = urpautils.read_txt_file("C:\\path\\to\\file.txt")
+
+# Read json file. Returns FileNotFoundError if the file does not exist
```
Comment (anerold) [inline README.md:48]: again, "Raises"

File: urpautils/file_utils.py [hunk 3]
```diff
@@ -105,6 +106,17 @@ def read_txt_file(file_name: str, encoding: str = "utf-8") -> str:
         return txt_file.read()
 
 
+def read_json_file(file_name: str, encoding: str = "utf-8") -> str:
```
Comment (anerold) [inline urpautils/file_utils.py:109]: `json.load()` returns `dict`, not `str`

**Review by anerold — CHANGES_REQUESTED** [review]

**Comment by anerold** [comment]
Also "check with Black" action is failing. Probably due to indented docstring of the `read_json` function

File: README.md (hunk 1 above)
Comment (Tscunami) [reply README.md]: Fixed it.

File: README.md (hunk 2 above)
Comment (Tscunami) [reply README.md]: Fixed it.

File: urpautils/file_utils.py (hunk 3 above)
Comment (Tscunami) [reply urpautils/file_utils.py]: Fixed it.

**Comment by Tscunami** [comment]
Each issue mentioned above should be fixed in the 0431251 commit.

File: urpautils/file_utils.py [hunk 4]
```diff
@@ -105,6 +106,17 @@ def read_txt_file(file_name: str, encoding: str = "utf-8") -> str:
         return txt_file.read()
 
 
+def read_json_file(file_name: str, encoding: str = "utf-8") -> dict:
+    """Reads a json file
+
+    :param file_name:    path
+    :param encoding:     encoding to use
+    :return:             str content
```
Comment (anerold) [inline urpautils/file_utils.py:114]: One last thing - this now should be `:return:       dict content`

After this fix I'm ready to merge your pull request

Comment (Tscunami) [reply urpautils/file_utils.py]: Sorry, I should've changed that too. It's fixed in the 66dc8a3 commit.

**Pull request merged**